sapi/fpm/fpm/fpm_main.c

```diff
@@ -109,6 +109,10 @@ int __riscosify_control = __RISCOSIFY_STRICT_UNIX_SPECS;
 #include "fpm_log.h"
 #include "zlog.h"
 
+#if HAVE_SIGNAL_H
```

Comment:
Notice that `HAVE_SIGNAL_H` has been removed in 7.4 5f8915786f

Reply:
Fixed in the rebase...